app/src/main/res/layout-land/activity_main.xml

```diff
@@ -0,0 +1,206 @@
+
+    <androidx.constraintlayout.helper.widget.Flow
+        android:id="@+id/flow_social_links"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        app:constraint_referenced_ids="button_facebook,button_linkedIn,button_instagram"
+        app:flow_firstHorizontalBias="0.5"
+        app:flow_firstHorizontalStyle="spread_inside"
+        app:flow_horizontalGap="32dp"
+        app:layout_constraintBottom_toTopOf="@id/button_edit_profile"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/top_view" />
+
+    <androidx.appcompat.widget.AppCompatImageButton
+        android:id="@+id/button_facebook"
+        style="@style/ImageButtonStyle"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:contentDescription="@string/facebook_description"
+        app:layout_constraintBottom_toBottomOf="@id/flow_social_links"
+        app:layout_constraintTop_toTopOf="@id/flow_social_links"
+        app:srcCompat="@drawable/facebook" />
+
+    <androidx.appcompat.widget.AppCompatImageButton
+        android:id="@+id/button_linkedIn"
+        style="@style/ImageButtonStyle"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:contentDescription="@string/linkedin_description"
+        app:layout_constraintBottom_toBottomOf="@id/flow_social_links"
+        app:layout_constraintTop_toTopOf="@id/flow_social_links"
+        app:srcCompat="@drawable/linkedin" />
+
+    <androidx.appcompat.widget.AppCompatImageButton
+        android:id="@+id/button_instagram"
+        style="@style/ImageButtonStyle"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:contentDescription="@string/instagram_description"
+        app:layout_constraintBottom_toBottomOf="@id/flow_social_links"
+        app:layout_constraintTop_toTopOf="@id/flow_social_links"
+        app:srcCompat="@drawable/instagram" />
+
+    <de.hdodenhof.circleimageview.CircleImageView
+        android:id="@+id/icon_profile"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:src="@drawable/icon1"
+        app:layout_constraintBottom_toBottomOf="@id/top_view"
+        app:layout_constraintDimensionRatio="1:1"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintVertical_bias="0.15"
+        app:layout_constraintHorizontal_bias="0.5"
+        app:layout_constraintWidth_percent="0.10"/>
+
+    <androidx.constraintlayout.helper.widget.Flow
+        android:id="@+id/flow_name_career"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        android:insetTop="0dp"
```

Comment:
I think you are not going to need changed insets in flow. Or I do not understand what they do. Remove insets from all flows on all layouts if they really are redundant to these flows.

Reply:
I don't really understand what you mean. I hope this is what is needed.